src/python/pants/bin/pants_runner.py

```diff
@@ -57,9 +57,9 @@ def _should_run_with_pantsd(self, global_bootstrap_options: OptionValueContainer
 
     @staticmethod
     def scrub_pythonpath() -> None:
-        # Do not propagate any PYTHONPATH that happens to have been set in our environment
+        # Don't propagate any PYTHONPATH that happens to have been set in our environment
```

Comment:
This is getting rather arbitrary. Why remove the "it's" above but add the "don't" contraction here?